src/extension/statusbartoggler.ts

```diff
@@ -43,23 +47,41 @@ export class StatusBarToggler implements Disposable {
         this.update();
     }
 
+    public setCoverage(linePercentage: number | undefined ) {
+        if (Number.isFinite(linePercentage)) {
```

Comment:
[F] This let's us differentiate between "No Coverage" (i.e there is no coverage information for this file) and "Coverage 0%" (i.e there is coverage but 0 hits for this file).

Reply:
**Question:** Just for my understanding what do you mean when you add a `[F]` at the front?
I got the others (Q for question, etc) but I was curious about this one haha.